### ISSUE TITLE
Getting number from the paper

### ISSUE THREAD

**Issue by robertmccraith** [Zengyi-Qin/Weakly-Supervised-3D-Object-Detection#8]
Hi, I followed the steps in the README with the intention of getting the predictions on the kitti validation set but currently I get <10% using the pre-trained weights.
To evaluate I use the kitti object evaluation script with the threshold set to 0.5 for cars and restricting the files read to only the validation set.
Is there any tricks to getting this to work or advice on how you run evaluation on the outputs?
The values in the output files look incorrect compared to the GT.

**Comment by Zengyi-Qin** [comment]
In `evaluate_object.cpp` of your kitti evaluation code, was the const double N_SAMPLE_PTS in line 59 set to 41 or 11? It seems that the current cpp file that KITTI evaluation website provides sets the constant to 41. But previously the the constant was set to 11 officially and the KITTI website clarified this modification. It was a long time ago when we did the experiments and we used N_SAMPLE_PTS = 11. I would suggest to set the constant to 11 and recompile the evaluation code, and see if this is the problem.
